<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.940362, y1:0.894, x2:1, y2:0, stop:0 rgba(131, 246, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="gatelist">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>131</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
    background-color: white;
    border: 2px solid rgb(140, 200, 255);
    border-radius: 15px;
    padding: 6px;
    font-family: Futura;
    font-size: 14px;
    color: black;
}
QListWidget::item {
    padding: 8px;
    border-radius: 10px;
}
QListWidget::item:hover {
    background-color: rgb(220, 235, 255);
}
QListWidget::item:selected {
    background-color: rgb(140, 200, 255);
    color: white;
}
</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="circuitTableWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>651</width>
      <height>331</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    background-color: white;
    border: 2px solid rgb(140, 200, 255);
    border-radius: 15px;
    gridline-color: rgb(200, 230, 255);
    font-family: Futura;
    font-size: 14px;
    color: black;
}

QHeaderView::section {
    background-color: rgb(140, 200, 255);
    color: white;
    font-family: Futura;
    font-weight: bold;
    padding: 6px;
    border: none;
}

QTableWidget::item {
    padding: 8px;
    border-radius: 8px;
}

QTableWidget::item:selected {
    background-color: rgb(140, 200, 255);
    color: white;
}
</string>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="runbtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>540</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(140, 200, 255);
    color: white;
    border-radius: 20px;  
    padding: 6px 12px;  
    border: none;  
}

QPushButton:hover {
    background-color: rgb(155, 186, 255); 
}
</string>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetbtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>620</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(140, 200, 255);
    color: white;
    border-radius: 20px;  
    padding: 6px 12px;  
    border: none;  
}

QPushButton:hover {
    background-color: rgb(155, 186, 255); 
}
</string>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hintbtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>670</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: rgb(140, 200, 255);
    color: white;
    border-radius: 20px;  
    padding: 6px 12px;  
    border: none;  
}

QPushButton:hover {
    background-color: rgb(155, 186, 255); 
}
</string>
    </property>
    <property name="text">
     <string>HINT</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultTextEdit">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>170</y>
      <width>131</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
    background-color: white;
    border: 2px solid rgb(140, 200, 255);
    border-radius: 15px;
    padding: 8px 10px;
    font-family: Futura;
    font-size: 14px;
    color: black;
}

QTextEdit:focus {
    border: 2px solid rgb(155, 186, 255); 
}
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Futura'; font-size:14px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'.AppleSystemUIFont'; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="questionLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>971</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    border: 2px solid rgb(140, 200, 255);
    border-radius: 20px;
    padding: 6px 10px;
    font-family: Futura;
    font-size: 14px;
    color: black;
}

QLineEdit:focus {
    border: 2px solid rgb(155, 186, 255); /* Lighter blue when focused */
}
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="circuitLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>971</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    border: 2px solid rgb(140, 200, 255);
    border-radius: 20px;
    padding: 6px 10px;
    font-family: Futura;
    font-size: 14px;
    color: black;
}

QLineEdit:focus {
    border: 2px solid rgb(155, 186, 255); /* Lighter blue when focused */
}
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
